yt/odin/checks/bin/unaware_nodes/__main__.py

```diff
@@ -1,9 +1,26 @@
 from yt_odin_checks.lib.check_runner import main
 
+DEFAULT_ENABLED_NODE_FLAVORS = [
+    "data",
+    "exec",
+    "tablet",
+    "chaos"
+]
 
-def find_unaware_nodes(yt_client):
+
+def find_unaware_nodes(yt_client, check_node_flavors):
     result = []
-    for row in yt_client.list("//sys/cluster_nodes", attributes=["rack"], read_from="cache"):
+    for row in yt_client.list("//sys/cluster_nodes", attributes=["rack", "flavors"], read_from="cache"):
+        flavors = row.attributes.get("flavors", [])
+
+        check_node_rack = False
+        for node_flavor in flavors:
+            if node_flavor in check_node_flavors:
+                check_node_rack = True
+                break
+        if not check_node_rack and len(flavors) > 0:
```

Comment:
`len(flavors) > 0` -> `flavors`, it is a pythonic way to check list for non-emptyness